CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## (5.1.0 IN PROGRESS)
 
 * Add the "Is Donor" filter selectable options for filters pane. Refs UIPFO-44.
+* Create or add list of donors. Refs UIPFO-46.
```

Comment:
Please update the description according to the changes. Currently, it is not entirely clear what has changed in the module.

Reply:
And the PR's title as well